### DISCUSSION TITLE
DSAR REQUEST - Remove Billing Information

### DISCUSSION THREAD

**Discussion by Kaotic3** [community/community#132279] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If you can tell me how to do this, I will remove it myself. There doesn't appear to be a way to remove it, just exchange it for another payment choice, as such, I have had to make this a DSAR request.

Please accept this as a DSAR request to delete all billing information that you hold for this account.

You should be aware of European Legislation that still applies to the UK:
https://ico.org.uk/for-the-public/your-right-to-get-your-data-deleted/

**You don’t have to ask a specific person – you can contact any part of the organisation with your request.**

If you are unable to provide a way for me to delete the billing information myself, could you acknowledge receipt of my DSAR request and that this request is being passed to the correct people.

Thanks,

**Comment by ebndev** [comment]
Hi @Kaotic3,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
